### ISSUE TITLE
Support for Visual Studio 2019

### ISSUE THREAD

**Issue by ATTMetrologySolutionsAdamL** [nicollasricas/streamdeckvs#11]
This plugin "Enables Stream Deck integration with Visual Studio 2017, **2019** & 2022." but Command Explorer "Works with Visual Studio **2022** (amd64), 2022 (Arm64)". So I was wondering what is needed to use this with Visual Studio 2019.

![image](https://github.com/nicollasricas/streamdeckvs/assets/117410184/6b1f01bb-35e1-4328-a177-c37d7779524a)

![image](https://github.com/nicollasricas/streamdeckvs/assets/117410184/eabeee92-a87f-4022-b674-32ca69138197)

**Comment by EngstromJimmy** [comment]
No, that is just for finding the command names. You can also use keyboard shortcuts in Visual Studio to find them :)

**Comment by nicollasricas** [comment]
Hey,

It looks that the extension were updated and support for the 2019 version dropped.

I've compiled a list with the commands ID to test the new update I'm working on, which will add support to Loupedeck and also a lot of new features.

[names.pdf](https://github.com/nicollasricas/streamdeckvs/files/11617722/names.pdf) - [Virus Total Scan](https://www.virustotal.com/gui/file-analysis/NTQ4NmI4ZGRmY2VjNDczMTM4ZWRmNDA1NzNkOWYyY2Q6MTY4NTU1NzcwNw==)

Hope it helps.
